definitions/contested/json/CaseField/CaseField.json

```diff
@@ -2677,6 +2677,14 @@
     "FieldType": "Label",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "12/05/2021",
+    "CaseTypeID": "FinancialRemedyContested",
+    "ID": "consentOrderFRCText",
```

Comment:
You will need to allocate permissions for users to see this field

Reply:
:white_check_mark: